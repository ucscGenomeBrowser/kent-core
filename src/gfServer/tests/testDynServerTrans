#!/bin/bash -e

testName=$(basename $0)

# build index
gfRootDir=output/${testName}_root
gfDir=${gfRootDir}/crea
rm -rf $gfDir
mkdir -p $gfDir
cp -f input/creaGeno.2bit $gfDir/crea.2bit
gfServer index -trans $gfDir/crea.trans.gfidx $gfDir/crea.2bit >/dev/null

# test query

./dynTester $gfRootDir transQuery crea crea input/crea.mrna output/${testName}.ver.out
sed -e 's/^version .*$/version XXX/' output/${testName}.ver.out > output/${testName}.out
diff -u expected/${testName}.out output/${testName}.out
