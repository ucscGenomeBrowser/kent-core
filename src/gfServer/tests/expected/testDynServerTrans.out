0ddf270562684f29transQuery crea crea 1540

tileSize 4
0	104	crea.2bit:hCreaGeno	80	184	23	+	0	0
 0 80 8 88 12 92 16 96 20 100 24 104 28 108 32 112 36 116 44 124 48 128 52 132 56 136 60 140 64 144 68 148 72 152 76 156 80 160 84 164 88 168 96 176 100 180
309	345	crea.2bit:hCreaGeno	868	904	6	+	0	0
 309 868 313 872 321 880 325 884 333 892 341 900
179	199	crea.2bit:hCreaGeno	528	548	4	+	0	0
 179 528 183 532 191 540 195 544
366	382	crea.2bit:mCreatGeno	2092	2108	3	+	0	0
 366 2092 374 2100 378 2104
395	415	crea.2bit:mCreatGeno	2176	2196	3	+	0	0
 395 2176 399 2180 411 2192
433	505	crea.2bit:hCreaGeno	1780	1852	15	+	0	1
 433 1780 437 1784 441 1788 453 1800 457 1804 461 1808 465 1812 469 1816 473 1820 477 1824 481 1828 485 1832 493 1840 497 1844 501 1848
107	163	crea.2bit:hCreaGeno	388	444	14	+	0	1
 107 388 111 392 115 396 119 400 123 404 127 408 131 412 135 416 139 420 143 424 147 428 151 432 155 436 159 440
106	162	crea.2bit:mCreatGeno	1104	1160	11	+	0	1
 106 1104 110 1108 114 1112 118 1116 122 1120 126 1124 130 1128 134 1132 150 1148 154 1152 158 1156
203	275	crea.2bit:hCreaGeno	680	752	12	+	0	2
 203 680 211 688 219 696 223 700 227 704 231 708 235 712 239 716 243 720 247 724 255 732 271 748
393	433	crea.2bit:hCreaGeno	1672	1712	9	+	0	2
 393 1672 397 1676 401 1680 405 1684 409 1688 413 1692 417 1696 425 1704 429 1708
348	388	crea.2bit:hCreaGeno	1460	1500	8	+	0	2
 348 1460 352 1464 356 1468 364 1476 368 1480 376 1488 380 1492 384 1496
284	304	crea.2bit:hCreaGeno	804	824	5	+	0	2
 284 804 288 808 292 812 296 816 300 820
54	74	crea.2bit:mCreatGeno	856	876	3	+	0	2
 54 856 58 860 70 872
202	274	crea.2bit:hCreaGeno	680	752	18	+	1	0
 202 680 206 684 210 688 214 692 218 696 222 700 226 704 230 708 234 712 238 716 242 720 246 724 250 728 254 732 258 736 262 740 266 744 270 748
347	391	crea.2bit:hCreaGeno	1460	1504	11	+	1	0
 347 1460 351 1464 355 1468 359 1472 363 1476 367 1480 371 1484 375 1488 379 1492 383 1496 387 1500
392	432	crea.2bit:hCreaGeno	1672	1712	10	+	1	0
 392 1672 396 1676 400 1680 404 1684 408 1688 412 1692 416 1696 420 1700 424 1704 428 1708
308	344	crea.2bit:mCreatGeno	1608	1644	9	+	1	0
 308 1608 312 1612 316 1616 320 1620 324 1624 328 1628 332 1632 336 1636 340 1640
53	89	crea.2bit:mCreatGeno	856	892	7	+	1	0
 53 856 57 860 61 864 65 868 69 872 73 876 85 888
279	303	crea.2bit:hCreaGeno	800	824	6	+	1	0
 279 800 283 804 287 808 291 812 295 816 299 820
0	100	crea.2bit:hCreaGeno	80	180	24	+	1	1
 0 80 4 84 8 88 12 92 16 96 24 104 28 108 32 112 36 116 40 120 44 124 48 128 52 132 56 136 60 140 64 144 68 148 72 152 76 156 80 160 84 164 88 168 92 172 96 176
205	273	crea.2bit:mCreatGeno	1424	1492	17	+	1	1
 205 1424 209 1428 213 1432 217 1436 221 1440 225 1444 229 1448 233 1452 237 1456 241 1460 245 1464 249 1468 253 1472 257 1476 261 1480 265 1484 269 1488
305	345	crea.2bit:hCreaGeno	864	904	10	+	1	1
 305 864 309 868 313 872 317 876 321 880 325 884 329 888 333 892 337 896 341 900
350	390	crea.2bit:mCreatGeno	2076	2116	10	+	1	1
 350 2076 354 2080 358 2084 362 2088 366 2092 370 2096 374 2100 378 2104 382 2108 386 2112
395	431	crea.2bit:mCreatGeno	2176	2212	8	+	1	1
 395 2176 399 2180 403 2184 407 2188 411 2192 419 2200 423 2204 427 2208
171	199	crea.2bit:hCreaGeno	520	548	7	+	1	1
 171 520 175 524 179 528 183 532 187 536 191 540 195 544
174	202	crea.2bit:mCreatGeno	1228	1256	7	+	1	1
 174 1228 178 1232 182 1236 186 1240 190 1244 194 1248 198 1252
103	171	crea.2bit:hCreaGeno	384	452	17	+	1	2
 103 384 107 388 111 392 115 396 119 400 123 404 127 408 131 412 135 416 139 420 143 424 147 428 151 432 155 436 159 440 163 444 167 448
433	509	crea.2bit:hCreaGeno	1780	1856	17	+	1	2
 433 1780 437 1784 441 1788 445 1792 449 1796 453 1800 457 1804 461 1808 465 1812 473 1820 481 1828 485 1832 489 1836 493 1840 497 1844 501 1848 505 1852
102	170	crea.2bit:mCreatGeno	1100	1168	17	+	1	2
 102 1100 106 1104 110 1108 114 1112 118 1116 122 1120 126 1124 130 1128 134 1132 138 1136 142 1140 146 1144 150 1148 154 1152 158 1156 162 1160 166 1164
276	304	crea.2bit:mCreatGeno	1536	1564	7	+	1	2
 276 1536 280 1540 284 1544 288 1548 292 1552 296 1556 300 1560
436	460	crea.2bit:mCreatGeno	2348	2372	6	+	1	2
 436 2348 440 2352 444 2356 448 2360 452 2364 456 2368
432	508	crea.2bit:hCreaGeno	1780	1856	16	+	2	0
 432 1780 436 1784 448 1796 452 1800 456 1804 460 1808 464 1812 468 1816 476 1824 480 1828 484 1832 488 1836 492 1840 496 1844 500 1848 504 1852
106	170	crea.2bit:hCreaGeno	388	452	14	+	2	0
 106 388 110 392 114 396 122 404 126 408 130 412 134 416 142 424 146 428 150 432 154 436 158 440 162 444 166 448
105	169	crea.2bit:mCreatGeno	1104	1168	12	+	2	0
 105 1104 109 1108 113 1112 121 1120 125 1124 129 1128 133 1132 149 1148 153 1152 157 1156 161 1160 165 1164
202	274	crea.2bit:hCreaGeno	680	752	15	+	2	1
 202 680 206 684 210 688 214 692 218 696 222 700 226 704 230 708 234 712 238 716 242 720 254 732 258 736 266 744 270 748
347	387	crea.2bit:hCreaGeno	1460	1500	9	+	2	1
 347 1460 351 1464 355 1468 359 1472 363 1476 371 1484 375 1488 379 1492 383 1496
392	428	crea.2bit:hCreaGeno	1672	1708	8	+	2	1
 392 1672 396 1676 404 1684 408 1688 412 1692 416 1696 420 1700 424 1704
279	303	crea.2bit:hCreaGeno	800	824	5	+	2	1
 279 800 283 804 291 812 295 816 299 820
53	73	crea.2bit:mCreatGeno	856	876	3	+	2	1
 53 856 57 860 69 872
0	100	crea.2bit:hCreaGeno	80	180	23	+	2	2
 0 80 4 84 8 88 12 92 16 96 20 100 24 104 28 108 32 112 36 116 40 120 44 124 52 132 56 136 60 140 64 144 68 148 72 152 80 160 84 164 88 168 92 172 96 176
205	273	crea.2bit:mCreatGeno	1424	1492	7	+	2	2
 205 1424 209 1428 217 1436 241 1460 253 1472 261 1480 269 1488
305	345	crea.2bit:hCreaGeno	864	904	6	+	2	2
 305 864 317 876 325 884 333 892 337 896 341 900
354	386	crea.2bit:mCreatGeno	2080	2112	5	+	2	2
 354 2080 358 2084 370 2096 374 2100 382 2108
171	191	crea.2bit:hCreaGeno	520	540	4	+	2	2
 171 520 179 528 183 532 187 536
end
0ddf270562684f29transQuery crea crea 1606

tileSize 4
452	528	crea.2bit:mCreatGeno	2348	2424	16	+	0	0
 452 2348 456 2352 464 2360 468 2364 472 2368 476 2372 480 2376 484 2380 488 2384 492 2388 496 2392 500 2396 508 2404 512 2408 516 2412 524 2420
122	186	crea.2bit:mCreatGeno	1104	1168	14	+	0	0
 122 1104 126 1108 130 1112 138 1120 142 1124 146 1128 150 1132 158 1140 162 1144 166 1148 170 1152 174 1156 178 1160 182 1164
123	187	crea.2bit:hCreaGeno	388	452	12	+	0	0
 123 388 127 392 131 396 139 404 143 408 147 412 151 416 167 432 171 436 175 440 179 444 183 448
296	320	crea.2bit:mCreatGeno	1540	1564	5	+	0	0
 296 1540 300 1544 308 1552 312 1556 316 1560
1	117	crea.2bit:mCreatGeno	788	904	26	+	0	1
 1 788 5 792 9 796 13 800 17 804 25 812 29 816 33 820 37 824 41 828 45 832 49 836 57 844 61 848 65 852 69 856 73 860 77 864 81 868 85 872 89 876 93 880 97 884 101 888 109 896 113 900
321	361	crea.2bit:mCreatGeno	1604	1644	8	+	0	1
 321 1604 325 1608 333 1616 337 1620 341 1624 345 1628 353 1636 357 1640
223	291	crea.2bit:hCreaGeno	684	752	6	+	0	1
 223 684 227 688 235 696 259 720 271 732 287 748
372	396	crea.2bit:hCreaGeno	1468	1492	4	+	0	1
 372 1468 376 1472 388 1484 392 1488
413	433	crea.2bit:hCreaGeno	1676	1696	3	+	0	1
 413 1676 425 1688 429 1692
222	290	crea.2bit:mCreatGeno	1424	1492	13	+	0	2
 222 1424 226 1428 230 1432 234 1436 238 1440 242 1444 246 1448 250 1452 258 1460 270 1472 274 1476 278 1480 286 1488
408	448	crea.2bit:mCreatGeno	2172	2212	9	+	0	2
 408 2172 412 2176 420 2184 424 2188 428 2192 432 2196 436 2200 440 2204 444 2208
367	403	crea.2bit:mCreatGeno	2076	2112	7	+	0	2
 367 2076 371 2080 375 2084 387 2096 391 2100 395 2104 399 2108
187	219	crea.2bit:mCreatGeno	1224	1256	6	+	0	2
 187 1224 191 1228 199 1236 203 1240 211 1248 215 1252
68	88	crea.2bit:hCreaGeno	132	152	3	+	0	2
 68 132 72 136 84 148
322	358	crea.2bit:hCreaGeno	864	900	3	+	0	2
 322 864 350 892 354 896
229	289	crea.2bit:mCreatGeno	1432	1492	13	+	1	0
 229 1432 233 1436 237 1440 241 1444 245 1448 249 1452 253 1456 257 1460 261 1464 265 1468 273 1476 281 1484 285 1488
362	406	crea.2bit:mCreatGeno	2072	2116	9	+	1	0
 362 2072 366 2076 370 2080 378 2088 382 2092 390 2100 394 2104 398 2108 402 2112
411	447	crea.2bit:mCreatGeno	2176	2212	7	+	1	0
 411 2176 415 2180 419 2184 423 2188 427 2192 431 2196 443 2208
71	87	crea.2bit:hCreaGeno	136	152	3	+	1	0
 71 136 75 140 83 148
452	528	crea.2bit:mCreatGeno	2348	2424	18	+	1	1
 452 2348 456 2352 460 2356 468 2364 472 2368 476 2372 480 2376 484 2380 488 2384 492 2388 496 2392 500 2396 504 2400 508 2404 512 2408 516 2412 520 2416 524 2420
122	178	crea.2bit:mCreatGeno	1104	1160	14	+	1	1
 122 1104 126 1108 130 1112 134 1116 138 1120 142 1124 146 1128 150 1132 154 1136 158 1140 162 1144 166 1148 170 1152 174 1156
123	179	crea.2bit:hCreaGeno	388	444	11	+	1	1
 123 388 127 392 131 396 135 400 139 404 143 408 147 412 151 416 167 432 171 436 175 440
300	320	crea.2bit:mCreatGeno	1544	1564	4	+	1	1
 300 1544 304 1548 308 1552 316 1560
1	117	crea.2bit:mCreatGeno	788	904	25	+	1	2
 1 788 5 792 9 796 17 804 25 812 29 816 33 820 37 824 41 828 45 832 49 836 53 840 61 848 65 852 69 856 73 860 77 864 81 868 85 872 89 876 93 880 97 884 101 888 105 892 113 900
325	361	crea.2bit:mCreatGeno	1608	1644	7	+	1	2
 325 1608 329 1612 337 1620 341 1624 345 1628 349 1632 357 1640
227	291	crea.2bit:hCreaGeno	688	752	6	+	1	2
 227 688 235 696 247 708 259 720 271 732 287 748
409	433	crea.2bit:hCreaGeno	1672	1696	4	+	1	2
 409 1672 413 1676 425 1688 429 1692
372	396	crea.2bit:hCreaGeno	1468	1492	3	+	1	2
 372 1468 384 1480 392 1488
0	116	crea.2bit:mCreatGeno	788	904	25	+	2	0
 0 788 4 792 8 796 12 800 16 804 28 816 32 820 36 824 40 828 44 832 52 840 56 844 64 852 68 856 72 860 76 864 80 868 84 872 88 876 92 880 96 884 100 888 104 892 108 896 112 900
218	290	crea.2bit:hCreaGeno	680	752	18	+	2	0
 218 680 222 684 226 688 230 692 234 696 238 700 242 704 246 708 250 712 254 716 258 720 262 724 266 728 270 732 274 736 278 740 282 744 286 748
367	407	crea.2bit:hCreaGeno	1464	1504	10	+	2	0
 367 1464 371 1468 375 1472 379 1476 383 1480 387 1484 391 1488 395 1492 399 1496 403 1500
324	360	crea.2bit:mCreatGeno	1608	1644	9	+	2	0
 324 1608 328 1612 332 1616 336 1620 340 1624 344 1628 348 1632 352 1636 356 1640
412	448	crea.2bit:hCreaGeno	1676	1712	8	+	2	0
 412 1676 416 1680 420 1684 424 1688 428 1692 436 1700 440 1704 444 1708
295	319	crea.2bit:hCreaGeno	800	824	6	+	2	0
 295 800 299 804 303 808 307 812 311 816 315 820
221	289	crea.2bit:mCreatGeno	1424	1492	17	+	2	1
 221 1424 225 1428 229 1432 233 1436 237 1440 241 1444 245 1448 249 1452 253 1456 257 1460 261 1464 265 1468 269 1472 273 1476 277 1480 281 1484 285 1488
362	406	crea.2bit:mCreatGeno	2072	2116	11	+	2	1
 362 2072 366 2076 370 2080 374 2084 378 2088 382 2092 386 2096 390 2100 394 2104 398 2108 402 2112
321	361	crea.2bit:hCreaGeno	864	904	10	+	2	1
 321 864 325 868 329 872 333 876 337 880 341 884 345 888 349 892 353 896 357 900
407	447	crea.2bit:mCreatGeno	2172	2212	10	+	2	1
 407 2172 411 2176 415 2180 419 2184 423 2188 427 2192 431 2196 435 2200 439 2204 443 2208
67	103	crea.2bit:hCreaGeno	132	168	8	+	2	1
 67 132 71 136 75 140 79 144 83 148 87 152 91 156 99 164
186	218	crea.2bit:mCreatGeno	1224	1256	8	+	2	1
 186 1224 190 1228 194 1232 198 1236 202 1240 206 1244 210 1248 214 1252
191	215	crea.2bit:hCreaGeno	524	548	6	+	2	1
 191 524 195 528 199 532 203 536 207 540 211 544
119	187	crea.2bit:hCreaGeno	384	452	17	+	2	2
 119 384 123 388 127 392 131 396 135 400 139 404 143 408 147 412 151 416 155 420 159 424 163 428 167 432 171 436 175 440 179 444 183 448
118	186	crea.2bit:mCreatGeno	1100	1168	17	+	2	2
 118 1100 122 1104 126 1108 130 1112 134 1116 138 1120 142 1124 146 1128 150 1132 154 1136 158 1140 162 1144 166 1148 170 1152 174 1156 178 1160 182 1164
452	528	crea.2bit:mCreatGeno	2348	2424	17	+	2	2
 452 2348 456 2352 460 2356 464 2360 468 2364 472 2368 476 2372 480 2376 488 2384 496 2392 500 2396 504 2400 508 2404 512 2408 516 2412 520 2416 524 2420
449	477	crea.2bit:hCreaGeno	1780	1808	7	+	2	2
 449 1780 453 1784 457 1788 461 1792 465 1796 469 1800 473 1804
292	320	crea.2bit:mCreatGeno	1536	1564	7	+	2	2
 292 1536 296 1540 300 1544 304 1548 308 1552 312 1556 316 1560
end
0ddf270562684f29transInfo crea crea

version XXX
serverType dynamic
type translated
tileSize 4
stepSize 4
minMatch 3
end
